### PR TITLE
Text obscure update

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: src/components/text/_text.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@ $sgTextDarkSemitransparentColor: rgba(0, 0, 0, 0.4);
     }
 
     &--obscure {
-      @include typeVariant(obscure, 1);
+      @include typeVariant(obscure, 0.5);
```
Comment (kdzwinel) [inline src/components/text/_text.scss]: not 0.75?

Comment (aju) [inline src/components/text/_text.scss]: it 0.5 of rhythm so 0.75.rem so 12 px :D

**Comment by vergilius** [comment]
OK

**Comment by kdzwinel** [comment]
ok

**Pull request merged**